views/allShapes.ejs

```diff
@@ -0,0 +1,67 @@
+<!DOCTYPE html>
+<html>
+  <title>UFO Sightings</title>
+  <link
+    rel="stylesheet"
+    href="https://cdn.jsdelivr.net/npm/bootstrap@4.5.3/dist/css/bootstrap.min.css"
+    integrity="sha384-TX8t27EcRE3e/ihU7zmQxVncDAy5uIKz4rEkgIXeMed4M0jlfIDPvg6uqKI2xXr2"
+    crossorigin="anonymous"
+  />
+  <link rel="stylesheet" href="/styles.css" />
+
+  <!--NavBar-->
+  <nav class="navbar navbar-expand-sm fixed-top navbar-light navBarBg">
+    <a class="navbar-brand">UFO Sightings!</a>
+
+    <!-- Toggler/collapsibe Button -->
+    <button
+      class="navbar-toggler"
+      type="button"
+      data-toggle="collapse"
+      data-target="#collapsibleNavbar"
+    >
+      <span class="navbar-toggler-icon"></span>
+    </button>
+    <!-- Navbar links -->
+    <div class="collapse navbar-collapse" id="collapsibleNavbar">
+      <ul class="navbar-nav">
+        <li class="nav-item active">
+          <a class="nav-link" href="/">Home</a>
+        </li>
+        <li class="nav-item">
+          <a class="nav-link" href="/shapes">Shapes</a>
+        </li>
+        <li class="nav-item">
+          <a class="nav-link" href="/newSighting">Report New Sighting</a>
+        </li>
+      </ul>
+    </div>
+  </nav>
+
+  <body>
+    <div class="container navTopMargin">
+      <div class="pageTitle">Sighting List of Shapes</div>
+      <ol>
+        <% sightings.forEach((sighting,index)=>{ %> 
+          <%Object.entries(sighting).forEach(([key,value])=>{ %> 
+            <%if(key==='shape'){ %>
+            <li><a href="/shapes/<%=value%>"><%=value%></a></li>
+            <% } %> 
+          <% }) %>
+        <br />
+        <% }) %>
+      </ol>
+    </div>
+  </body>
+
+  <script
+    src="https://code.jquery.com/jquery-3.5.1.slim.min.js"
+    integrity="sha384-DfXdz2htPH0lsSSs5nCTpuj/zy4C+OGpamoFVy38MVBnE+IbbVYUew+OrCXaRkfj"
+    crossorigin="anonymous"
+  ></script>
+  <script
```

Comment:
Do we need to include Bootstrap via script, or can we include it via `npm` and `import` now?